main.cpp

```diff
@@ -76,45 +103,73 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
     return {}; //if your function returns something other than void, add 'return {};' at the end of it.
 } 
 
-/*
- 1)
- */
+ float carCost(float downPayment, float carPrice, int numCars)
+ {
+     ignoreUnused(downPayment, carPrice, numCars = 1);
+     return {};
+ }
 
-/*
- 2)
- */
 
-/*
- 3)
- */
+  char password(char password, char username)
+ {
+     ignoreUnused(password, username);
+     return {};
+ }
+ 
 
-/*
- 4)
- */
 
-/*
- 5)
- */
+  void moveArm(int direction, int distance)
+ {
+     ignoreUnused(direction, distance);
+ }
+ 
 
-/*
- 6)
- */
+ void moveLeg(int distance, int direction)
+ {
+     ignoreUnused(distance, direction);
+ }
+ 
 
-/*
- 7)
- */
+  bool powerOn(bool switchOn, bool powerFlow)
+ {
+     ignoreUnused(switchOn, powerFlow);
+     return {};
+ }
+ 
 
-/*
- 8)
- */
+ void car(int direction, int speed)
+ {
+     ignoreUnused(direction, speed = 55);
+ }

```

Comment:
- `car` is not an action, and is a bad name for a function.
Name things what they are or what they do.
- you are assigning the value `55` to `speed`.
This is not demonstrating providing default values to function arguments.
- [ ] revise